rero_ils/static/js/rero_ils/rero_keep_tab.js

```diff
@@ -0,0 +1,36 @@
+/*
+
+RERO ILS
+Copyright (C) 2020 RERO
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, version 3 of the License.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+*/
+
+(function ( $ ) {
+  $.fn.keeptab = function( options ) {
+    var opts = $.extend( {
+      localeItem: 'navkeeptab',
+      selector: '.nav-link'
+    }, options );
+    $('#' + this.attr('id') + ' ' + opts.selector).on('click', function() {
```

Comment:
This should be non relevant if `this` doesn't have an `id`. 
Maybe better to use ``` this.find(opts.selected).on(....) ``` ?

Reply:
I changed this with a new parameter.